packages/core/components/QuerySidebar/QuerySidebar.module.css

```diff
@@ -81,30 +112,27 @@
     white-space: nowrap;
 }
 
-.container > * {
-    margin-bottom: 5px;
-}
-
 .queries-container {
     height: calc(100% - var(--header-height) - var(--footer-height));
     overflow-y: scroll;
+    padding-bottom: 40px;
 }
 
 .queries-container::after {
     background-image: linear-gradient(transparent, var(--secondary-sidebar-color));
-    bottom: var(--footer-height);
+    bottom: calc(var(--footer-height) + var(--margin) / 2);
     content: " ";
     height: 50px;
     left: 0;
     max-width: var(--query-sidebar-max-width);
     position: absolute;
-    width: calc(0.3 * 80% - var(--margin));
+    width: 100%;
     z-index: 2;
 }
 
 .minimize-bar {
```

Comment:
Can't tell if there's some padding missing on the minimize bar or if it's just the scrollbar being weird
<img width="149" alt="image" src="https://github.com/AllenInstitute/aics-fms-file-explorer-app/assets/24441706/74ca360f-3765-4063-b627-6e80ed7e004e">

Reply:
This will be removed / altered in part 2 majorly